### ISSUE TITLE
Generating type IDs

### ISSUE THREAD

**Issue by drewdepriest** [google/digitalbuildings#193]
When adding new YAML object types, how do you generate the 19-20 digit ID string?

That every id attribute being divisible by 4 would seem to imply it's auto-generated, though I cannot find any such auto-enumerator or other schema within the repo. 

Does the validator generate said id string on some kind of merge trigger?

Cheers,
Drew

**Comment by berkoben** [comment]
Yes, there should be tooling to generate a GUID.  We have it in our backlog to add.

**Comment by charbull** [comment]
@drewdepriest for now keep it empty string please, we will populate that later.

**Comment by mschulze17** [comment]
@berkoben & @charbull - should we add this to the backlog?

**Comment by charbull** [comment]
@kswang2400 is working on it in the current sprint
